go/oasis-test-runner/scenario/e2e/txsource.go

```diff
@@ -76,17 +79,32 @@ func (sc *txSourceImpl) Fixture() (*oasis.NetworkFixture, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Use deterministic identities as we need to allocate funds to nodes.
+	f.Network.DeterministicIdentities = true
 	f.Network.StakingGenesis = "tests/fixture-data/txsource/staking-genesis.json"
 
 	// Disable CheckTx on the client node so we can submit invalid transactions.
 	f.Clients[0].ConsensusDisableCheckTx = true
 
-	// MinGasPrice
-	f.Validators = []oasis.ValidatorFixture{
-		oasis.ValidatorFixture{Entity: 1, MinGasPrice: 1},
-		oasis.ValidatorFixture{Entity: 1, MinGasPrice: 1},
-		oasis.ValidatorFixture{Entity: 1, MinGasPrice: 1},
```

Comment:
how did this work before, was `SubmissionGasPrice` by default 1?

Reply:
validators accepted their own tx'es (no fee's required), other nodes failed registrations but none of the workload tests tested anything runtime related so no test failed :-)

Reply:
ok